solidity/random-beacon/test/RandomBeacon.Pool.test.ts

```diff
@@ -4,34 +4,42 @@ import { expect } from "chai"
 import { Signer, Contract } from "ethers"
 import { randomBeaconDeployment } from "./fixtures"
 
-import type { RandomBeacon, SortitionPoolStub } from "../typechain"
+import type { RandomBeacon, SortitionPool, StakingStub } from "../typechain"
+
+const fixture = async () => randomBeaconDeployment(undefined)
```

Comment:
No need to pass `undefined` explicitly once this param is optional.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2684/commits/625a2862af7bad90fb3d5bfd83a8ff861e901892.